resque-scheduler.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+// Look for an environment variable with 
+$RESQUE_PHP = getenv('RESQUE_PHP');
+if (!empty($RESQUE_PHP)) {
+	require_once $RESQUE_PHP;
+}
+// Otherwise, if we have no Resque then assume it is in the include path
+else if (!class_exists('Resque')) {
+	require_once 'Resque/Resque.php';
+}
+
+// Load resque-scheduler
+require_once dirname(__FILE__) . '/lib/ResqueScheduler.php';
+require_once dirname(__FILE__) . '/lib/ResqueScheduler/Worker.php';
```

Comment:
We should be able to replace this with the Composer autoloader.

Reply:
We've done exactly this on https://github.com/M2Mobi/php-resque/pull/1